core/services/pipeline/runner_test.go

```diff
@@ -164,6 +166,7 @@ func TestRunner(t *testing.T) {
 		for _, run := range runs {
 			if run.DotID() == "ds1" {
 				assert.True(t, run.Error.IsZero())
+				require.NotNil(t, resp, run.Output)
```

Comment:
Could we also assert somewhere that the AllowUnrestrictedNetworkAccess on the HTTPTask is actually set to true from the DOT source?

Reply:
I could not figure out an easy or obvious way to test this.

Perhaps @spooktheducks has some ideas

Reply:
fwiw I ended up doing something similar here https://github.com/smartcontractkit/chainlink/pull/3698/files#diff-090f788b3a083e95e45cee241250eeb3cb0215d4b553c2737753634ab9ef4b79R12